Switch pipeline to new review app system (#601)
mozilla/network-pulse-api: README.md

### Review App

#### Review App for PRs

Opening a PR will automatically create a Review App in the `network-pulse-api` pipeline. A slack bot posts credentials and links to Review Apps in to the `mofo-ra-pulse-api` Slack channel.

*Note:* This only work for Mo-Fo staff: you will need to manually open a Review App on Heroku for PRs opened by external contributors.

#### Review App for branches

You can manually create a review app for any branch pushed to this repo. It's useful if you want to test your code on Heroku without opening a PR yet.

To create one:
- log into Heroku.

> - Go to the `foundation-site` pipeline.

Reviewer: ```suggestion
- Go to the `network-pulse-api` pipeline.
```

- Click on `+ New app` and select the branch you want to use.

> The review app slack bot will post a message in the `foundation-site` with links and credentials as soon as the review app is ready.

Reviewer: ```suggestion
The review app slack bot will post a message in the `mofo-ra-pulse-api` channel with links and credentials as soon as the review app is ready.
```

#### Environment variables:

- `GITHUB_TOKEN`: GITHUB API authentication,

> - `SLACK_WEBHOOK_RA`: Webhook to `mofo-ra-foundation`

Reviewer: ```suggestion
- `SLACK_WEBHOOK_RA`: Webhook to `mofo-ra-pulse-api`
```
